v2/spiffetls/mode.go

```diff
@@ -40,7 +46,8 @@ type dialMode struct {
 }
 
 type listenMode struct {
-	tlsType    authMode
+	mode       serverMode
+	raw        bool
```

Comment:
nit: for someone looking at this for the first time this may not be super obvious, we may add a comment explaining what is this.

Reply:
i renamed the variable and added a comment.